Assets/Shaders/Caustics.gdshader

```diff
@@ -46,7 +45,7 @@ mediump vec2 panner(mediump vec2 uv, mediump float speed, mediump float tiling){
 	return (vec2(1, 0) * TIME * speed) + (uv * tiling);
 }
 
-mediump vec3 aberration_sample(sampler2D _CausticsTexture, mediump vec2 uv, mediump float split){
+mediump vec3 aberration_sample(sampler2D _CausticsTexture, mediump vec2 uv, float split){
```

Comment:
Removing the single `mediump` here fixed a visual bug where the chromatic aberration was very pixelated.